coding-standard/README.md

```diff
@@ -27,4 +27,7 @@ jobs:
       with:
         version: 25 # Optional, will use the latest if omitted.
         path: app/code # Optional, will be used when event is not a pull request.
+        severity: 8 # Optional, will use phpcs default of 5 if not specified.
```

Comment:
Actually, I was looking at this @Vinai can you add a test workflow covering with/without these flags? I'm not 100% confident that those conditionals work the way you expect (they very well may, my bash is just weak).

Reply:
I tested it in the MR and then removed those commits from the history.
That said, I'd like to add a test workflow, but I'm not sure how to do that.
Can we have a chat about it in Discord maybe?